client_src/js/services/axios.service.js

```diff
@@ -0,0 +1,169 @@
+import axios from "axios";
+import { ClientError } from "../exceptions/client-error.handler";
+import { ServerError } from "../exceptions/server-error.handler";
+import { PrinterClientError } from "../exceptions/printer-client-error.handler";
+import { HTTPError } from "../exceptions/api.exceptions";
+import { PrinterApiErrors } from "../exceptions/printer-api.exceptions";
+import { ClientErrors } from "../exceptions/client.exceptions";
+import { validateServerResponse, validatePath } from "../utils/validators/api.validator";
+import AmIAliveService from "./amialive.service";
+
+function octofarmErrorResponseCheck(response) {
+  switch (response.status) {
+    case 0:
+      throw new ServerError(HTTPError.NO_CONNECTION, response.data);
```

Comment:
No because to me the OctoPrint errors we are going to receive require a more user based response than 404/402/500 or whatever. This is something I'm still working out but currently OP errors are going to show in a notification as before and OctoFarm errors will trigger the full pop-up. OctoPrint errors can be stuff like "CONFLICT" which if we responded to the user that, it doesn't make much sense. They need to work similar to how they did before within the new error checking framework as they're are now using the same. OP can't trigger the same as OF.